### PR TITLE
refact: removed Identity types + dpp

### PR DESCRIPTION
**Pull request by Alex-Werner**
### Issue being fixed or implemented  

With @dashevo/js-dpp release 0.12 that is soon to be released, the Identity types has been removed. 
We had a dependency on DPP on two elements : 
- DPA (2019 feature that has been left unused)
- Identity (with type support). 
Therefore our dependency on DPP is not justified anymore on wallet-lib and can be safely removed.

### What was done  

- Removed DPP 
- Removed DPA Plugin feature (was unused and outdated feature; pre-DashJS)
- Updated documentation

### How Has This Been Tested?

Using our current tests that covers identity. 

### Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/types/Account/Account.d.ts [hunk 1]
```diff
@@ -26,8 +26,6 @@ export declare class Account {
     getUnconfirmedBalance(displayDuffs?:boolean): number;
     getBIP44Path(network?:Network, index?:number): string;
 
-    getDPA(name: string): object;
-
     getIdentityHDKey(index: number, identityType: string): HDPrivateKey;
```
Comment (shumkov) [inline src/types/Account/Account.d.ts:29]: Still Identity type?

File: src/types/Account/methods/getIdentityHDKey.js [hunk 2]
```diff
@@ -1,21 +1,21 @@
-const Identity = require('@dashevo/dpp/lib/identity/Identity');
-
 /**
  * Returns a private key for managing an identity
  * @param {number} index - index of an identity
  * @param {string} identityType - type of identity (user, application)
```
Comment (shumkov) [inline src/types/Account/methods/getIdentityHDKey.js:4]: Update JSDoc

File: src/types/Account/Account.d.ts (hunk 1 above)
Comment (Alex-Werner) [reply src/types/Account/Account.d.ts:29]: Oh ! Thank you, I've missed this one X.X

File: src/types/Account/methods/getIdentityHDKey.js (hunk 2 above)
Comment (Alex-Werner) [reply src/types/Account/methods/getIdentityHDKey.js]: Fixed

**Pull request merged**